shared-libs/server-checks/test/checks.js

```diff
@@ -199,12 +214,16 @@ describe('Server Checks service', () => {
         },
         exit: () => 0
       };
+      sinon.stub(http, 'get').callsArgWith(1, { statusCode: 401 });
```

Comment:
What this text actually not working before?  (But just was being missed since it did not fail if `service.check` succeeded)